pkg/diagnostics/grpc_tracing.go

```diff
@@ -33,36 +33,39 @@ import (
 )
 
 const (
-	GRPCTraceContextKey = "grpc-trace-bin"
-	GRPCProxyAppIDKey   = "dapr-app-id"
-	daprPackagePrefix   = "/dapr.proto"
+	GRPCTraceContextKey     = "grpc-trace-bin"
+	GRPCProxyAppIDKey       = "dapr-app-id"
+	daprInternalPrefix      = "/dapr.proto.internals."
+	daprRuntimePrefix       = "/dapr.proto.runtime."
```

Comment:
and why is the `daprPackagePrefix` being split into two? 
can there be any other cases other than these two(leaving aside service invocation)?

Reply:
We need to be able to distinguish calls from other Dapr sidecars (internals - primarily, but not only, service invocation) from calls from apps to Dapr (runtime).

Reply:
This may change a bit once the streaming version of CallLocal is implemented by the way.